randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -344,10 +349,14 @@ Extra - asset_id: 10
 > Door to Landing Site; Heals? False; Default Node
   * Layers: default
   * Normal Door to Landing Site/Door to Crateria Power Bomb Room
+  > Pickup (Power Bomb Expansion)
+      Trivial
```

Comment:
can you add a TODO here for the requirement to get past the dragons(?) in your way?

Reply:
Right. A weapon or enough health to get through.

Reply:
I checked, and dragons can be killed with power beam. Nothing more needed here.